lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -908,6 +911,34 @@ function dosomething_campaign_set_est_timezone($date, $add_four = FALSE, $end_of
 }
 
 /**
+ * Returns array of date times ofor the given campaign node
+ *
+ * @param Object or Array
+ *    vars array or node object to pull from
+ *
+ * @return array $timing
+ *    today - The current day
+ *    start_date - The day the campaign starts
+ *    end_date - The day the campaign ends
+ */
+function dosomething_campaign_get_goals_times($node) {
+  if (!isset($node)) {
+    return NULL;
+  }
+  date_default_timezone_set('EST');
+  $high_season = is_array($node) ? $node['field_high_season'] : $node->field_high_season[LANGUAGE_NONE];
```

Comment:
Is the hot module still working on Thor/Prod? Something isnt right then because the existing logic doesn't specify language